### PR TITLE
Onboarding slide order change + copy updates

### LINKED ISSUE #7037: Swap slide 1 & 3
_TASK_
Resort the order of onboarding so the reportbacks are displayed first w/ a welcome message and additional campaign signups are shown last.

### LINKED ISSUE #7038: Update Copy on Slide 1
_TASK_
Now that slide one shows reportback photos, lets make sure all copy is updated to reflect our current brand.

### PR DESCRIPTION
**Pull request by itsjoekent**
#### What's this PR do?
- Changes the order of onboarding slides
- Updates the copy on all of them
#### How should this be reviewed?

Do the copy updates look ok? Dynamic variables fill in properly?
Is the new slide order correct?
#### Any background context you want to provide?

no
#### Relevant tickets

Fixes #7038 
Fixes #7037

### CODE REVIEW

File: lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc [hunk 1]
```diff
@@ -325,22 +325,22 @@ function dosomething_helpers_add_js_onboarding($nid) {
...
-    }
-
     if (variable_get('dosomething_campaign_onboarding_campaign_slide', FALSE)) {
       $onboardingSettings['slides'][] = 'CampaignSlide';
     }
 
+    if (variable_get('dosomething_campaign_onboarding_context_slide', FALSE)) {
+      $onboardingSettings['slides'][] = 'ContextSlide';
+    }
+
```
Comment (weerd) [inline lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc:343]: Why did you shuffle these?

File: lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js [hunk 2]
```diff
@@ -1,8 +1,7 @@
 const React = require('react');
 
-const title = "This is how you do it.";
-const body = "We are PUMPED to see your photos. Complete these simple steps and upload a photo of yourself in action. Your pics will inspire others to join the movement \u2013 so don't forget to share them!";
-
+const title = "Upload photos of yourself in action!";
+const bodies = ["DoSomething.org staff reviews every single photo you send, and we’ll feature our favorites in the gallery on the site.", "We are PUMPED to see your photos. Complete these simple steps of yourself in action. Your pics will inspire others to join the movement -- upload photos and watch the likes roll in!"];
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js]: `bodies` sounds weird. Could we rename to `content` maybe and then in the loop refer to it as `paragraph` or something...? Nit-picky. Whatever you change it to, change it in the **CampaignSlide.js** & **ReportbackItemsSlide.js** too.

Just a thought...

**Review by weerd — CHANGES_REQUESTED** [review]
Quick comments.

File: lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc (hunk 1 above)
Comment (itsjoekent) [reply lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc:343]: Thats what the issue asked for..?

File: lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js (hunk 2 above)
Comment (itsjoekent) [reply lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js]: sure

File: lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc (hunk 1 above)
Comment (weerd) [reply lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc:343]: But why not just reshuffle [here](https://github.com/DoSomething/phoenix/blob/96e1346d63e92d63068889d845b6664b6c76b860/lib/themes/dosomething/paraneue_dosomething/js/onboarding.js#L56)? Isn't this where it actually matters? Or am I confusing things...?

Comment (itsjoekent) [reply lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc:343]: OH thats what you meant, thats where i originally tried but it didn't work. We're mapping from the ds settings list, which means the order of of slideComponents is irrelevant

Comment (weerd) [reply lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc:343]: Oh gotcha! I thought it worked for me, but maybe that was also cause I might have needed to maybe comment some out and it was just a coincidence it worked right. I feel there's a better way than shuffling code around like that, lol. But ok.

File: lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 const React = require('react');
 
 const title = "Upload photos of yourself in action!";
-const bodies = ["DoSomething.org staff reviews every single photo you send, and we’ll feature our favorites in the gallery on the site.", "We are PUMPED to see your photos. Complete these simple steps of yourself in action. Your pics will inspire others to join the movement -- upload photos and watch the likes roll in!"];
+const content = ["DoSomething.org staff reviews every single photo you send, and we’ll feature our favorites in the gallery on the site.", "We are PUMPED to see your photos. Complete these simple steps of yourself in action. Your pics will inspire others to join the movement -- upload photos and watch the likes roll in!"];
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js]: Sorry. Forgot to mention this. Mind reformatting when it's an array so it's more obvious what the paragraphs are? One per line.

``` javascript
const content = [
  "DoSomething.org staff reviews every single photo you send, and we’ll feature our favorites in the gallery on the site.", 
  "We are PUMPED to see your photos. Complete these simple steps of yourself in action. Your pics will inspire others to join the movement -- upload photos and watch the likes roll in!"
];
```

Comment (itsjoekent) [reply lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js]: oh sure thing

**Review by weerd — APPROVED** [review]

**Comment by weerd** [comment]
¡OLÉ! :sheep: it!

**Pull request merged**